tests/Zend/TranslateTest.php

```diff
@@ -731,11 +732,13 @@ public function testIgnoreMultipleDirectories()
             'auto',
             array(
                 'scan' => Zend_Translate::LOCALE_FILENAME,
-                'ignore' => array('.', 'regex_1' => '/de_DE/', 'regex' => '/ja/')
+                'ignore' => array('.', 'regex_1' => '/de_DE/', 'regex' => '/translation-ja/'),
```

Comment:
This regex `/ja/` actually caused the correct file to be ignored on my system (since my username, which is in the path of the file, begins with "ja"), just making the regex more specific.